src/app/clarity/descriptionInterface.ts

```diff
@@ -0,0 +1,46 @@
+export interface LinesContent {
+  text?: string;
+  className?: string;
+  linkUrl?: string;
+  linkText?: string;
+  title?: string;
+}
+export interface Line {
+  lineText?: LinesContent[];
+  className?: string;
+}
+type PerkType =
+  | 'armorExotic'
+  | 'armorMod'
+  // ---------
+  | 'weaponPerkExotic'
+  | 'weaponFrameExotic'
+  | 'weaponPerk'
+  | 'weaponPerkEnhanced'
+  | 'weaponFrame'
+  | 'weaponMod'
+  | 'weaponCatalystExotic'
+  // ---------
+  | 'ghostMod';
+
+interface Perk {
+  id: number;
+  name: string;
+
+  itemId?: number;
```

Comment:
Is this the inventory item hash?

Reply:
Yes and
Id: perk hash
ItemId: armor hash
I can add more info if needed

Reply:
It's always nice to document those properties just so we know what they refer to.

Reply:
Yeah I realized if you have to ask it should be documented